olm/olm-template/manifests/rhoas-operator.cloudservicesrequests.crd.yaml

```diff
@@ -84,5 +84,25 @@ spec:
                         type: string
                       status:
                         type: string
+                schemaRegistries:
```

Comment:
This will be breaking change so version should be changed

Reply:
I think you can add without breaking things; I was going to confirm this in the k8s docs.